.run-clang-format

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+find . -type d \( -name bullet3_ros -o -name fcl_ros -o -name libccd_ros -o -name gtest_ros -o -name tesseract_ext \) -prune -o -type f -regex '.*\.\(cpp\|hpp\|cc\|cxx\|h\|hxx\)' -exec clang-format-8 -style=file -i {} \;
```

Comment:
Change to exclude Tesseract-specific package names

Reply:
I changed this to a dummy package name because I did not want to break the script, and I thought it might be something we will need if we have to bundle anything external into yak

Reply:
Are there any other dependencies needed to run the Clang formatting script? I.e., installing other libraries?

Reply:
I suppose you have to have clang-format-8 installed.

Reply:
On melodic you should be able to `sudo apt install clang-format-8`